wordpress-mobile/gutenberg-mobile
# Start playback immediately after video goes full screen

Fixes wordpress-mobile/gutenberg-mobile#1074

This is an iOS only issue.

gutenberg PR https://github.com/WordPress/gutenberg/pull/15998

To test:

- Tap on a video
- Verify that full screen player opens in a playing state

Update release notes:
